spring-integration-smb/src/main/java/org/springframework/integration/smb/filters/SmbPersistentAcceptOnceFileListFilter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
```

Comment:
This is a new class in this year, so must be only `2018`.

All other affected classes must be changes to reflect the current year in their second number.

Plus all the affected classes should have your name now! :smile: